member.php

```diff
@@ -2832,9 +2832,16 @@
 	}
 
 	$adminoptions = '';
-	if($mybb->usergroup['cancp'] == 1 && $mybb->config['hide_admin_links'] != 1)
+	if($mybb->usergroup['cancp'] == 1 && $mybb->config['hide_admin_links'] != 1 && (!$memban['uid'] || $memban['uid'] && ($mybb->user['uid'] == $memban['admin'])))
```

Comment:
This condition is not correct. This will prevent other admins (apart from the banning admin) even see the admin options in the banned member's profile.